packages/app/src/cli/models/extensions/specifications/transform/app_config_webhook.ts

```diff
@@ -1,87 +1,46 @@
-import {WebhookSubscription, WebhooksConfig} from '../types/app_config_webhook.js'
+import {WebhooksConfig, NormalizedWebhookSubscription} from '../types/app_config_webhook.js'
 import {deepMergeObjects, getPathValue} from '@shopify/cli-kit/common/object'
 
 export function transformWebhookConfig(content: object) {
-  const webhooks = getPathValue<WebhooksConfig>(content, 'webhooks')
+  const webhooks = getPathValue(content, 'webhooks') as WebhooksConfig
   if (!webhooks) return content
 
-  // normalize webhook config with the top level config
   const webhookSubscriptions = []
-  const {topics, subscriptions, uri, api_version: apiVersion} = webhooks
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  const {api_version, subscriptions = []} = webhooks
 
-  if (uri && topics?.length) {
-    for (const topic of topics) {
-      webhookSubscriptions.push({
-        topic,
-        uri,
-      })
-    }
+  // eslint-disable-next-line no-warning-comments
+  // TODO: pass along compliance_topics once we're ready to store them in its own module
+  for (const {uri, topics, compliance_topics: _, ...data} of subscriptions) {
```

Comment:
> opted to spread (...) the rest of the subscription params in case we ever support more, so we won't need to update all the parsing logic and only the schema + module (only the module in the future I guess once CLI as a client is a thing)

Makes sense, it also gets simpler. But I've renamed data to optionalFields, which sounds more descriptive to me. Is it ok?